resources/views/auth/first_visit_manager.html.twig

```diff
@@ -0,0 +1,97 @@
+{# =============================================================================
+
+    Talent Cloud
+    Auth: Register
+
+============================================================================= #}
+
+{% extends "layouts/base" %}
+
+{% block title %}{{ first_visit.title }}{% endblock %}
+
+{% block body %}
+    {% include "common/goc" %}
+    <div data-clone>
+        <section class="tc-auth" data-c-padding="tb(triple)" data-c-overlay="black(80)" style="background-image: url('/images/bg_crowd.jpg');">
+            <div class="tc-auth-wrapper" data-c-padding="tb(triple)" data-c-container="small">
+                {# Return to the homepage link. #}
+                <a data-c-color="white" href="/" title="Return Home">
+                    <i class="fas fa-chevron-left" data-c-margin="right(quarter)"></i>{{ first_visit.return }}
+                </a>
+                <div data-c-radius="rounded" data-c-margin="top(normal)">
+                    {# Auth UI Title Area #}
+                    <div class="tc-auth-header gradient-left-right" data-c-padding="all(double)" data-c-align="base(center)">
+                        <h1 data-c-font-size="h2" data-c-color="white">{{ first_visit.header }}</h1>
+                    </div>
+                    {# Registration Form #}
+                    <form action="{{ routes.continue }}" method="POST">
+                        {{ csrf_field() }}
+                        <div data-c-background="white(100)" data-c-padding="tb(normal) rl(double)">
+                            <div data-c-padding="top(normal)">
+                                {% if errors.any() %}
+                                    {# Clone alert box for errors. #}
+                                    <div
+                                        data-c-alert="error"
+                                        data-c-radius="rounded"
+                                        role="alert" data-c-margin="bottom(double)">
+                                        <div data-c-padding="half">
+                                            <span
+                                                data-c-heading="h5"
+                                                data-c-font-weight="bold">
+                                                <i aria-hidden="true" class="fas fa-exclamation-circle"></i>
+                                                Uh oh, something doesn't look right.
```

Comment:
This line and line 45 need to be localized

Reply:
Localized and extracted into an error component.